XML/TestCases/FunctionalTests-SGX.xml

```diff
@@ -0,0 +1,15 @@
+<TestCases>
+    <test>
+        <testName>VALIDATE-INTEL-SGX-DRIVER-FOR-DC-VM</testName>
+        <testScript>validate-intel-sgx-driver.sh</testScript>
+	<timeout>1800</timeout>
+        <setupType>OneVM</setupType>
+        <OverrideVMSize>Standard_DC2s</OverrideVMSize>
```

Comment:
Suggest we work off the new GA VM SKUs in uksouth
```suggestion
        <OverrideVMSize>Standard_DC2s_v2</OverrideVMSize>
```

Reply:
No change, Re-open

Reply:
Fixed.